index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const fr = require('./lib/reader.js');
+
+module.exports = exports = {};
+
+exports.fileRunner = (err, data) => {
+  const filesNames = ['one.txt', 'two.txt', 'three.txt'];
+  filesNames.forEach((ele) => {
```

Comment:
This does not guarantee that they will complete in order. If the first takes longer then the 2nd it could be different. They will probably come back in order because on your computer it happens so fast that it won't be super noticeable and the speeds won't vary much. Also you are passing through ele, and err which the ele is the path so that is fine but the err is actually the callback from the test. While technically this is correct because fileReader wants the callback you shouldn't be calling it error because that isn't what it is.